work16/src/main.js

```diff
@@ -0,0 +1,298 @@
+      li.setAttribute("aria-selected", false);
+    }
+    li.setAttribute("roll", "tab");
+    li.id = `${"js_tabTopics" + ++newsCategoryIndex}`;
+    li.classList = "tabTopics";
+
+    a.textContent = newsCategoryObj.field
+    a.href = "#"
+
+    li.appendChild(a);
+    fragment.appendChild(li);
+  });
+  tabList.appendChild(fragment);
+
+  clickEventChangeTabAttribute();
+}
+
+const renderNewsContent = (newsDataArray) => {
+  newsDataArray.forEach((newsCategoryObj, newsCategoryIndex) => {
+    const tabTopics = document.querySelectorAll(".tabTopics");
+    const newsContentsData = newsCategoryObj.contents;
+    const newsCategoryImg = newsCategoryObj.img;
+    let isInitialDisplay = newsCategoryObj.isInitialDisplay;
+
+    if (isInitialDisplay === "true") {
+      const div = document.createElement("div");
+      const newsSection = createSection(tabTopics, newsCategoryIndex);
+
+      div.classList = "tabpanelTopics_wrap";
+
+      document.getElementById("js_tabList").after(newsSection);
+      div.appendChild(createTopicImg(newsCategoryImg));
+      div.prepend(renderArticle(newsContentsData, newsCategoryIndex));
+
+      newsSection.appendChild(div);
+    }
+
+    tabTopics[newsCategoryIndex].addEventListener("click", (e) => {
+      removeCommentIcon();
+      removeNewIcon();
+
+      newsContentsData.forEach((newsArticleData, categoryNewsArticleDataIndex) => {
+        const commentArray = newsArticleData.comments;
+        const liElements = document.querySelectorAll(".tabpanelTopics_wrap li");
+        const liElement = liElements[categoryNewsArticleDataIndex];
+
+        changeCategoryConetent(newsCategoryIndex, e.currentTarget);
+        changeCategoryTitle(newsArticleData, categoryNewsArticleDataIndex);
+        changeCategoryImg(newsCategoryImg);
+
+        if (commentArray.length !== 0) {
+          liElement.append(createCommentIcon(commentArray));
+        }
+
+        if (isLatestArticles(newsArticleData) === true) {
+          const h1Elements = document.querySelectorAll(".tabpanelTopics_wrap h1");
+          const h1Element = h1Elements[categoryNewsArticleDataIndex];
+          h1Element.insertAdjacentElement("afterend", createNewIcon());
+        }
+      })
+    })
+  })
+}
+
+const renderArticle = (newsContentsData) => {
+  const ul = document.createElement("ul");
+  const fragment = document.createDocumentFragment();
+
+  newsContentsData.forEach(newsArticleData => {
+    const li = document.createElement("li");
+    const h1 = document.createElement("h1");
+    const a = document.createElement("a");
+
+    const title = newsArticleData.title;
+    const commentArray = newsArticleData.comments;
+
+    a.href = "#";
+    a.textContent = title;
+
+    h1.appendChild(a)
+    li.appendChild(h1);
+
+    if (commentArray.length !== 0) {
+      li.append(createCommentIcon(commentArray));
+    }
+
+    if (isLatestArticles(newsArticleData) === true) {
+      h1.insertAdjacentElement("afterend", createNewIcon());
+    }
```

Comment:
IMO
```suggestion
    if (isLatestArticles(newsArticleData)) {
      h1.insertAdjacentElement("afterend", createNewIcon());
    }
```

Reply:
I learned that I can omit true in the conditional branch of an if statement.
Thank you very much. m(_ _)m

commit: [e58e83b](https://github.com/yasu-pro/js-handson/pull/18/commits/e58e83b07136394ca81dda8e3c990b38a2a5e36a)